### PR TITLE
Achecker 4 : Upgrading deprecated MySQL syntax to support MySQL 5. 6+

### PR DESCRIPTION
**Pull request by VictorAlagwu**
Changing the MySQL syntax to be compatible with MySQL 5.6+
Status: Still in progress

### CODE REVIEW

File: install/db/achecker_schema.sql [hunk 1]
```diff
@@ -308,7 +308,7 @@ CREATE TABLE `themes` (
   `title` varchar(80) NOT NULL DEFAULT '',
   `version` varchar(10) NOT NULL DEFAULT '',
   `dir_name` varchar(20) NOT NULL DEFAULT '',
-  `last_updated` date DEFAULT NOT NULL,
+  `last_updated` date NOT NULL,
```
Comment (cindyli) [inline install/db/achecker_schema.sql]: Please merge in the master that has the latest changes of your previous pull request. This change, as well as the change in `install/db/achecker_upgrade_1.4_to_1.5.sql` seems from an older version of your previous pull request.

Comment (VictorAlagwu) [reply install/db/achecker_schema.sql]: The changes only affected the ``` themes``` table

Comment (VictorAlagwu) [reply install/db/achecker_schema.sql]: If it is ok with you, I would send a different pull request to fix the issue

Comment (cindyli) [reply install/db/achecker_schema.sql]: Sure. Probably continue to use ACHECKER-3 ticket to fix this issue. Thanks.

Comment (VictorAlagwu) [reply install/db/achecker_schema.sql]: Ok, would do that.

**Pull request closed without merging**